transport/TransportImpl.cpp

```diff
@@ -2390,44 +2357,27 @@ void TransportImpl::runTick(uint64_t timestamp)
 
 void TransportImpl::doRunTick(const uint64_t timestamp)
 {
-    if (!_config.rctl.enable || !_config.bwe.useUplinkEstimate)
-    {
-        while (!_rtxPacingQueue.empty())
-        {
-            protectAndSendRtp(timestamp, _rtxPacingQueue.fetchBack());
-        }
-        while (!_pacingQueue.empty())
-        {
-            protectAndSendRtp(timestamp, _pacingQueue.fetchBack());
-        }
-        _pacingInUse = false;
-        return;
+    bool useBE = _config.rctl.enable && _config.bwe.useUplinkEstimate;
+    drainPacingBuffer(timestamp, useBE);
+    if (useBE) {
+        sendPadding(timestamp);
     }
+    _pacingInUse = !_pacingQueue.empty() || !_rtxPacingQueue.empty();
+}
 
-    auto budget = _rateController.getPacingBudget(timestamp);
-    while (!_pacingQueue.empty() || !_rtxPacingQueue.empty())
-    {
-        auto* pacingQueue = &_pacingQueue;
-        if (!_rtxPacingQueue.empty())
-        {
-            pacingQueue = &_rtxPacingQueue;
-        }
+inline memory::UniquePacket TransportImpl::tryFetchPriorityPacket(size_t budget)
```

Comment:
[minor] `inline` is not make any difference on this case on modern c++ compilers. Compiler in the end will decided inline it or not depend what it think is the best decision. In my opinion we should try to keep code as less verbose as possible and use `inline` when it is required

Reply:
Agree, since it's not "free function included from multiple files" it is unnecessary since. Removed inline spec.